wp-content/themes/core/components/blocks/content_loop/css/content_loop.pcss

```diff
@@ -0,0 +1,147 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Content Loop Header
```

Comment:
General comment: It would be nice to add header and `/* CASE: ... */` comments for all the style blocks ([like so](https://github.com/moderntribe/square-one/blob/main/wp-content/themes/core/components/blocks/card_grid/css/card_grid.pcss#L9-L14)).